man/man8/zpool-remove.8

```diff
@@ -50,7 +50,8 @@
 .Op Fl npw
 .Ar pool Ar device Ns ...
 .Xc
-Removes the specified device from the pool.
+Removes the specified device from the pool. Note that keys for all encrypted
```

Comment:
[nit] I think each sentence is supposed to start on a new line.

Reply:
Happy to fix! Will the squash be enough to satisfy your MR requirements? I recall a note about more than one commit being not allowed.

Reply:
> Will the squash be enough to satisfy your MR requirements?

Yes, you can either squash and force update the PR.  Or just add a commit and I'll squash it when merging.